knowledge/security/ir.model.access.csv

```diff
@@ -1,3 +0,0 @@
-id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_ir_attachment_sale_manager,ir_attachment manager,base.model_ir_attachment,base.group_sale_manager,1,1,1,1
-access_ir_attachment_partner_manager,ir_attachment_partner manager,base.model_ir_attachment,base.group_partner_manager,1,1,1,1

```

Comment:
Because:

* the groups didn't make any sense; why should attachments be restricted to sale managers and partner managers?
* the `base.group_sale_manager` group doesn't exist anymore, it has been moved to the `sales_team` module. I don't see why the `knowledge` module should depend on `sales_team`
* attachments already implement a custom logic for permissions: attachments linked to a record (so, all "user" attachments) can be accessed by the same people who can access the linked record. That logic is much better and in my opinion, sufficient.